### DISCUSSION TITLE
Rescheduled by frePPLe for "approved" orders

### DISCUSSION THREAD

**Discussion by Tska999** [frePPLe/frepple#443] [Q&A]
I had a purchase order and a production order. Bost had the status "approved". 
Now the documentation say: The manufacturing order is present in the ERP system but can still be rescheduled by frePPLe (optimization input).
The planing engine dont rescheduled these orders.
What can I do?
Can you help me

![image](https://user-images.githubusercontent.com/94317975/146921663-9fa7fdda-dedd-4611-a61b-e2d99c14e84e.png)

There is enough capacity

![image](https://user-images.githubusercontent.com/94317975/146921858-276396f7-0fd5-4359-ba08-c2d216b08ed5.png)

**Comment by jdetaeye** [comment]
The Community Edition doesn't have the capability to reschedule approved manufacturing orders.

Only the Enterprise and Cloud Editions will reschedule approved orders.  When approved orders are not feasible (i.e. planned in the past, missing material or overloading capacity) the algorithm will move approved to a new date where they are feasible.

>What can I do?

Buy the Enterprise or Cloud Edition :-)
